src/__tests__/__snapshots__/GreenButton.js.snap

```diff
@@ -41,7 +41,7 @@ exports[`GreenButton renders 1`] = `
   text-align: center;
   text-decoration: none;
   font-weight: 600;
-  cursor: pointer;
+  text-rendering: optimizeLegibility, -webkit-font-smoothing:antialiased cursor:pointer;
```

Comment:
Do you have an idea what is causing this?

Reply:
I do!  Fixed